visuals/MDS.py

```diff
@@ -0,0 +1,74 @@
+from votekit.profile import PreferenceProfile
+from metrics.distances import earth_mover_dist, Lp_dist
+import matplotlib.pyplot as plt
+import numpy as np
+from sklearn import manifold
+import sys
+
+sys.path.append("../")
+
+
+# Help function for MDS Plot
+def distance_matrix(
+    pp_set1: set[PreferenceProfile],
+    pp_set2: set[PreferenceProfile],
+    distance: str,
+    ballot_graph,
+):
+    """
+    creates pairwise distance matrix between preference profiles. The i-th and j-th entry are
+    pairwise distance between i-th col preference profile and the j-th row preference profile. \n
+    pp_set1 and pp_set2: set of preference profiles \n
+    distance: Use "L1" for L1 dist, "earth mover" for earth mover dist.
+    """
+    rows = len(pp_set1)
+    cols = len(pp_set2)
+    dist_matrix = np.zeros((rows, cols))
+    pp_arr1 = list(pp_set1)
+    pp_arr2 = list(pp_set2)
+
+    if distance == "L1":
```

Comment:
you might consider instead passing in a function (either Lp_dist or eearth_mover_dist) that would make this extensible to other distance measures without requiring string names of each one as you go